templates/Gemfile.erb

```diff
@@ -37,7 +37,8 @@ group :development, :test do
   gem "factory_girl_rails"
   gem "pry-byebug"
   gem "pry-rails"
-  gem "rspec-rails", "~> 3.4.0"
+  gem "refills"
+  gem "rspec-rails", "~> 3.5.0.beta4"
```

Comment:
It looks like there's going to be a release cut today - https://twitter.com/samphippen/status/748639259549114368